docs/articles/baseplatform/repo-manifests.rst

```diff
@@ -1,6 +1,67 @@
 Manifests
 =========
 
-Bla bla XML upstream, revision, where to put files and so on.
+Repo [#repotool]_ is a tool for using several git repositories in the same
+build. It features a `manifest` format using XML where one defines the git
+repositories and revisions one wants to use. It is possible to include manifests
+in other manifests.
 
-Structure, include other manifests etc.
+Feature overview
+----------------
+* Download several git repos simultaneously.
+* Use different revisions for different repos.
+
```

Comment:
Why do you have those empty lines in the list?

Reply:
Because that's how nested lists works in rst. You also need the empty line before the nest otherwise it thinks I'm doing a definition list (dl tag)